### ISSUE TITLE
AttributeError: 'StageModified' object has no attribute 'git'

### ISSUE THREAD

**Issue by Venique** [git-cola/git-cola#876]
Hi!

Trying to install git-cola at Windows 8. Lauch it from Start Menu, it works correctly until I'm trying to commit any file:

```
Traceback (most recent call last):
  File "C:\Program Files\git-cola\pkgs\cola\cmds.py", line 2376, in do_cmd
    return cmd.do()
  File "C:\Program Files\git-cola\pkgs\cola\cmds.py", line 2071, in do
    return Stage.do(self)
  File "C:\Program Files\git-cola\pkgs\cola\cmds.py", line 2003, in do
    return self.stage_paths()
  File "C:\Program Files\git-cola\pkgs\cola\cmds.py", line 2012, in stage_paths
    return self.stage_all()
  File "C:\Program Files\git-cola\pkgs\cola\cmds.py", line 2042, in stage_all
    status, out, err = self.git.add(v=True, u=True)
AttributeError: 'StageModified' object has no attribute 'git'
```

My environment:

```
Git Cola version 3.1 (v3.1) 
Windows-8.1-6.3.9600-SP0 
Python (C:\Program Files (x86)\git-cola\Python\pythonw.exe) 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:04:45) [MSC v.1900 32 bit (Intel)] 
Git 2.18.0.windows.1 
Qt 5.10.1 
QtPy 1.3.1 
PyQt5 5.10.1 
```

Even tried to downgrade to 3.0 but no luck. At first I have only git-cola 3.1 and git-for-windows 2.18 installed but after I've faced this error as you can see I've tried to downgrade, install full pack of software (python 3.3-3.6, pyqt4\5 etc), switch to my external installation of Python, not built-in:

```
Git Cola version 3.0 (v3.0-116-gdcc61a0e) 
Windows-8.1-6.3.9600-SP0 
Python (C:\Program Files\Python36\pythonw.exe) 3.6.6 (v3.6.6:4cf1f54eb7, Jun 27 2018, 03:37:03) [MSC v.1900 64 bit (AMD64)] 
Git 2.18.0.windows.1 
Qt 5.10.1 
QtPy 1.3.1 
PyQt5 5.10.1 
```

...but still no any positive result in any combination. What's wrong?

**Comment by davvid** [comment]
You'd probably be fine if you ran from the latest in master.  I'm going to release a new version very soon.  When that happens you should be able to use that version.

**Comment by davvid** [comment]
Just released v3.2.

With the latest installers, you don't need to install python, pyqt, etc. manually, but that should still work too either way.  They're still helpful to have installed if you want to hack on cola, or write PyQt5 apps.

As a curiosity -- if upgrading doesn't make a difference, try uninstalling first.  I really hope the Windows installer isn't crufty like that, but it's possible.  Please let me know if that's related at all.  That should be pretty easy to verify by checking the version numbers.

Thanks for the heads-up, cheers.

**Comment by Venique** [comment]
Installed 3.2, everything seems fine. Ty.
